www/include/monitoring/objectDetails/xml/serviceSendCommand.php

```diff
@@ -48,7 +49,8 @@
 
 CentreonSession::start(1);
 $centreon = $_SESSION["centreon"];
-if (!isset($_SESSION["centreon"]) ||
+if (
+    !isset($_SESSION["centreon"]) ||
     !isset($_GET["host_id"]) ||
     !isset($_GET["service_id"]) ||
     !isset($_GET["cmd"]) ||
```

Comment:
multiple isset can be done in one line, !isset($arg1,$arg2,$arg3)
```suggestion
    if ( !isset($_SESSION["centreon"], $_GET["host_id"], $_GET["service_id"], $_GET["cmd"], $_GET["actiontype"]))
```
The function isset while check the variable one by one and return false when it encounter a variable not set (same behaviour than using multiple OR operator)